### ISSUE TITLE
Make report accessible ... examples?

### ISSUE THREAD

**Issue by JNS728** [u-fischer/tagpdf#31]
I created a LaTeX-Template for a report which I have to submit rather soonly. The result is quite nice but I noticed (probably way too late) that, as of know, the final report won't be accessible for people who are challenged by PDF reports. This is a serious blow for me and I will not be able to use the template if I am unable to solve this problem.

The code of my template is available via the following link https://gitlab.tubit.tu-berlin.de/bunsen89/UBA-LaTeX

I read the `tagpdf` documentation but as a LaTeX beginner, it is quite difficult to understand what exatly I have to do.

I included `\include{uba/barrierefreiheit}` and added the following code.

```tex
\usepackage[luamode]{tagpdf}
\tagpdfsetup{activate-all}
```
I checked the accessibility of the resulting document with PAC3 and the result is still devastating. It does not fulfil any of the critria. Of course this is, because I've applied `tagpdf` in the wrong way. Probably it is more complicated than just adding to lines to my document. However, it did not become fully clear to me, how exactly this works. Therefore, I have two questions:

1) Are there some examples of reports (including the source code) to which I can refer and study the work steps?
2) Is it feasible for a beginner to turn the document which is compiled by the code behind the link above into an accessible document in a reasonable amount of time ( ... what's reasonable of course)? If it's too much work at this stage of the project, I may also just return to Word.

All answers, comments and remarks are much appreciated!
Thanks

**Comment by u-fischer** [comment]
The documentation itself is a proof of concept. It passes the pac3 test, There is a tagpdfdocu-patches.sty, which shows a number of changes I had to make to get it working, and the source code of the documentation shows the mark up. The code isn't the prettiest but I had only a few hours to do it. The docu doesn't show tabulars but here is an example https://github.com/u-fischer/tagpdf/issues/10. It doesn't show math either.  To correctly tag links you need a special hyperref driver which is developed in an experimental repo in the latex github. As it is in active development you would need from everything the head of the newest branches.  

To sum it up: Unlike other packages tagpdf *does* work. You can get a valid, properly tagged pdf with it. But it is not an easy-to-use-end-user-package. It is part of large project to improve latex in this area. I don't think that a beginner with time pressure should use it, better use word or use Adobe pro to tag your report. 

Side remark: one shouldn't use packages like sectsty or titlesec with a KOMA-class. And don't use \include in the preamble (and best use it only if you know why, normally \input is the better choice).

**Comment by sanojb99** [comment]
Thank you very much for your elaborate response! I know that tagpdf works because I ran the pac3 test on it. That's why I am so keen to use it :) However, it still seems that for me (as a beginner) maybe an application of tagpdf would be too challenging.
I'll check if I can retrofit my compiled document with Adobe Pro or if that's too much work given that my compiled document does not fulfil any of the pac3 test criteria. Anyway, your answer (as always) is very helpful and I appreciate your support and work!

I look forward to follow the development of this package. I'd be very eciting to see it being implemented in LaTeX as a standard at some point in the future.

Also many thanks for your side remarks! I'll try to solve them.
